Code/source/LexicalToken.cpp

```diff
@@ -0,0 +1,40 @@
+#include <string>
+
+using namespace std;
+
+#include "LexicalToken.h"
+
+//Check if string is a NAME Lexical. A Name Lexical starts with a alphabet and is followed by a string of alphanumerical
```

Comment:
Just a grammar issue but it should be NAME token. Lexical is a adjective

Reply:
I kept it as Lexical because it is both verification of INTEGER and NAME entity.